visual-tests/tests/cross-browser/focus.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { test } from '../../src/test-runner';
+import { helpers } from '../../src/helpers';
+
+test('Test focus on Shift+Tab navigation', async({ goto, tablePage }) => {
+  await goto('/basic-two-tables-demo');
+
+  const tableTop = tablePage.locator('#tableTop > .handsontable');
+  const tableBottom = tablePage.locator('#tableBottom > .handsontable');
+
+  tableTop.waitFor();
+  tableBottom.waitFor();
```

Comment:
`locator` and `waitFor` is a promises, shouldn't you wait for those? 

```suggestion
await  tableTop.waitFor();
await  tableBottom.waitFor();
```

Reply:
Correct, I found it in a few more places and also fixed it